student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(name, age, class_room)
+    super(name, age)
+    @class_room = class_room
```

Comment:
- [ ] Your `initialize` method is missing the `parent_permission` parameter. Kindly add it.
- [ ] The Student class inherits from Person hence it would it is recommended that you arrange your parameters to be more meaningful. In your Person `initialize` method age comes before name. Kindly interchange the positions of `age` and `name`

Reply:
I forgot to change the order here, thanks to Ted, I already fixed it!